integration/test/roster_test.js

```diff
@@ -0,0 +1,285 @@
+        });
+
+        it('`muted` of remote attendee is `false`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'muted', false);
+        });
+      });
+    });
+  });
+
+  describe('Video', async function () {
+    describe('enable', async function () {
+      before(async function () {
+        await localPage.toggleVideo();
+      });
+
+      describe('component', async function () {
+        it('local attendee has enabled video icon', async function () {
+          await localPage.checkVideoIcon(localAttendeeName, true);
+        });
+
+        it('remote attendee has enabled video icon', async function () {
+          await remotePage.checkVideoIcon(localAttendeeName, true);
+        });
+      });
+
+      describe('hook', async function () {
+        it('`videEnabled` of local attendee is `true`', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'videoEnabled', true);
+        });
+
+        it('`videEnabled` of remote attendee is `true`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'videoEnabled', true);
+        });
+      });
+    });
+
+    describe('disable', async function () {
+      before(async function () {
+        await localPage.toggleVideo();
+      });
+
+      describe('component', async function () {
+        it('local attendee has disabled video icon', async function () {
+          await localPage.checkVideoIcon(localAttendeeName, false);
+        });
+
+        it('remote attendee has disabled video icon', async function () {
+          await remotePage.checkVideoIcon(localAttendeeName, false);
+        });
+      });
+
+      describe('hook', async function () {
+        it('`videEnabled` of local attendee is `false`', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'videoEnabled', false);
+        });
+
+        it('`videEnabled` of remote attendee is `false`', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'videoEnabled', false);
+        });
+      });
+    });
+  });
+
+  describe('Content', async function () {
+    describe('start', async function () {
+      before(async function () {
+        await localPage.toggleContent();
+      });
+
+      describe('component', async function () {
+
+      });
+
+      describe('hook', async function () {
+        it('local attendee is sharing content', async function () {
+          await localPage.checkAttendeeState(localAttendeeName, 'sharingContent', true);
+        });
+
+        it('remote attendee is sharing content', async function () {
+          await remotePage.checkAttendeeState(localAttendeeName, 'sharingContent', true);
+        });
+      });
+    });
+
+    describe('stop', async function () {
+      before(async function () {
+        await localPage.toggleContent();
+      });
+      describe('component', async function () {
```

Comment:
Remove below?
```
describe('component', async function () {

      });
```

Reply:
I forgot adding the tests for component check 😅... Thanks for catching this

Reply:
Added and tested.